src/pages/landing-page/index.tsx

```diff
@@ -85,7 +88,7 @@ const LandingPage: React.FC = () => {
                         marginTop: '4rem',
                         marginBottom: '4rem',
                         gap: '3.5rem',
-                        backgroundColor: 'white',
+                        backgroundColor: '#f0f0f0',
```

Comment:
Why has this been changed even though it is not releted to the dark mode?

Reply:
Can we revert this change?

Reply:
we can do it. I pushed the commit

Reply:
Thanks @KitsuneKode!

Reply:
I got one more change to make

Reply:
done with the changes @krisstern